benchmark/cluster/vgg16/Dockerfile

```diff
@@ -1,18 +1,26 @@
-#FROM python:2.7.14
-FROM nvidia/cuda:8.0-cudnn5-runtime-ubuntu16.04
-RUN apt-get update && apt-get install -y python
+FROM paddlepaddle/paddle:latest-dev
```

Comment:
Do not use this image as base image, CI built image does not enable `WITH_DISTRIBUTED` yet.

Reply:
This not product image, it's the develop image.

Reply:
I see, but no need, `paddlepaddle/paddle:latest-dev` is rather large.

Reply:
Done.